### DISCUSSION TITLE
How to ignore the field in struct?

### DISCUSSION THREAD

**Discussion by TheGP** [diesel-rs/diesel#4348] [Q&A]
```
pub struct Token {
    pub public_key: String,
    pub name: Option<String>,
```

```
cannot find type `name` in module `crate::schema::tokens`
not found in `crate::schema::tokens`
```
Which macros to add to ignore?

**Comment by weiznich** [comment]
It's impossible to answer this question without details. Otherwise this would very similar to #860

**Comment by TheGP** [comment]
@weiznich  Yes, and I didnt see the answer in that topic. Why does your lib supports such a basic functionality? Why do u want me to change whole project to accomodate your lib? What a poorly designed project.

**Reply by weiznich** [reply]
If you expect future help here I suggest that you change you wording. We to not accept such wording here.

Otherwise for the technical reasons see #860. If you disagree with that you are open to use a different crate or to suggest a novel use-case that's **not** already listed in that issue (or implement the accepted subset of features and submit a PR).

**Reply by TheGP** [reply]
@weiznich you don't accept good ideas here as well. I have no intention to use such lib.
